tooling/bin/generate-final-artifact.sh

```diff
@@ -152,6 +152,33 @@ for architecture in "${architectures[@]}"; do
             "${tmp_folder_final_musl_appsec}/etc/recommended.json"
     fi
 
+    ########################
+    # PHP Stubs
+    ########################
+    stubs=(
+        "/home/circleci/datadog/src/ddtrace_php_api.stubs.php"
+        "/home/circleci/datadog/ext/ddtrace.stub.php"
+        "/home/circleci/datadog/ext/hook/uhook.stub.php"
+        "/home/circleci/datadog/ext/hook/uhook_attributes.stub.php"
+    )
+
+    mergedStubs=""
+    for stub in "${stubs[@]}"; do
+        content=$(<"$stub")
```

Comment:
What's that? The same than `content=$stub`?

Reply:
`$(<"$stub")` reads the contents of the file specified by "$stub" into the variable `content`. `stub` here will be one of the following:

```bash
(
        "/home/circleci/app/src/ddtrace_php_api.stubs.php"
        "/home/circleci/app/ext/ddtrace.stub.php"
        "/home/circleci/app/ext/hook/uhook.stub.php"
        "/home/circleci/app/ext/hook/uhook_attributes.stub.php"
)
```

I really am not a bash expert ^^ but I believe this is right 🤔 

I read the content of the stub file, remove the php handle, and then add the content to the global merged stubs.

Reply:
oh you are right :-) Never seen that specific syntax soup. I thought that was where the good old `cat` was needed, so disregard that.